flake.nix

```diff
@@ -459,24 +452,84 @@
           };
 
           devShells = rec {
-            developers = mkShell {
+            developers = mkShell (common-attrs // {
               inputsFrom = builtins.attrValues self.checks;
               buildInputs = with packages; [
                 # with nix developers are empowered for local dry run of most ci
-                rust-stable
+                llvmPackages_latest.llvm
+                llvmPackages_latest.bintools
+                lldb
+                zlib.out
+                xorriso
+                grub2
+                qemu
+                llvmPackages_latest.lld
+                openssl.dev
+                pkg-config
+                openssl
+                rust-nightly
                 wasm-optimizer
-                composable-node
+                rust-analyzer
                 mdbook
+                bacon
                 taplo
                 python3
                 nodejs
-                nixpkgs-fmt
+                nixpkgs-fmt 
+                rnix-lsp
                 jq
                 google-cloud-sdk # devs can list container images or binary releases
                 nix-tree
               ];
-              NIX_PATH = "nixpkgs=${pkgs.path}";
-            };
+                  
+              # RUSTC_VERSION = "stable";
```

Comment:
Remove all the comment?

Reply:
Accidentally opened PR before my cleanup commit, sorry